scan.py

```diff
@@ -189,6 +190,11 @@ def sns_start_scan(sns_client, s3_object, scan_start_sns_arn, timestamp):
 def sns_scan_results(
     sns_client, s3_object, sns_arn, scan_result, scan_signature, timestamp
 ):
+    # hack to disable false-positive https://github.com/Cisco-Talos/clamav/issues/620
+    if scan_result == AV_STATUS_INFECTED and scan_signature == "Archive.Test.Agent2-9953724-0 FOUND":
```

Comment:
What would be the best way to ingore such signatures? Shouldn't we create a separate file or DB for that?

Reply:
https://docs.clamav.net/manual/Signatures/AllowLists.html#signature-ignore-lists is the "right" way to do it